simplq/src/components/pages/Home/LandingPage.jsx

```diff
@@ -8,7 +8,11 @@ export default (props) => {
     <div className={styles['landing-page']}>
       <div>
         <Header className={styles['main-header']} text="SimplQ" />
-        <h3 className={styles.center}>A long overdue alternative to physical Queues</h3>
+        <p className={styles.subtitle}>A long overdue alternative to physical queues</p>
+        <p className={styles.description}>
+          Create and manage queues in instantly, free online queue management solution for easy
+          crowd control
+        </p>
```

Comment:
We can replace with a better text once finalised

Reply:
Maybe change it to 'Create and manage queues with our free, online queue management solution for easy and instant crowd control' for now.

Reply:
Done